dape.el

```diff
@@ -930,7 +930,10 @@ If NOWARN does not error on no active process."
          (json (json-serialize object :false-object nil))
          (string (format "Content-Length: %d\r\n\r\n%s" (length json) json)))
     (dape--debug 'io "Sending:\n%S" object)
-    (process-send-string process string)))
+    (condition-case err
```

Comment:
You might want `condition-case-unless-debug` here, else you look yourself out of useful backtraces if something does go wrong

Reply:
Ah nice one, had know idea of the existence of `condition-case-unless-debug`

Reply:
Reverted, ert enables debug on error which was the reson for it the first time around.

The handling of js-debug termination is not handled a clean way, there is no orphaned process but process is dies during termination with an strange timing.

For some reason one of the test does not work for emacs 28.2, which I am not motivated at this time to solve.

Reply:
Fair, and indeed ert enabling debug-on-error is a pain . You can rebind it in your tests tho. But you're right, this is soon to be replaced anyway.